README.md

```diff
@@ -57,7 +57,12 @@ man f3d # Linux only
 
 # Support
 
-F3D is developed by a team of passionate devs. Please use F3D, star it on github or even become a [sponsor](https://github.com/sponsors/f3d-app) to support us!
+F3D needs you help!
+
+If you can, please consider sponsoring F3D, even a small amount, it would help us offsetting the maintenance cost
+as well as considering adding support for devices we do not own yet. Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).
```

Comment:
```suggestion
If you can, please consider sponsoring F3D. Even a small donation would help us offsetting the recurring maintenance costs.
With enough sponsors we would even be able to add support for new devices (as we would need do acquire or rent them first). Read more about it on our [sponsor page](https://github.com/sponsors/f3d-app).
```
Clarify there's running costs, and you'd like to also be able to cover one-time costs (and what they would be, ie. access to hardware)